src/util-frontend.js

```diff
@@ -213,3 +213,22 @@ export function getToastErrorTimeout() {
 
     return errorTimeout;
 }
+
+/**
+ * Get debounced function
+ * @returns {Function} debounced function
+ */
+export function Debounce() {
```

Comment:
Function name should be camelcase no? Unless it's a class?

Reply:
https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Basics#introducing_constructors:~:text=Constructors%2C%20by%20convention%2C%20start%20with%20a%20capital%20letter